gadjid/src/graph_operations/ruletables/proper_ancestors.rs

```diff
@@ -65,7 +65,7 @@ mod test {
 
     use crate::PDAG;
 
-    use super::proper_ancestors;
+    use super::get_proper_ancestors;
 
     #[test]
     fn proper_ancestors_search() {
```

Comment:
Align across all `ruletables/*.rs` files so the test-naming is consistent – perhaps, `get_descendants` is the function with test `#[test] fn descendants` and analogously

Reply:
`get_x` test now just called `fn x() {..}` etc.